containers/Dockerfile.fedora

```diff
@@ -1,19 +1,21 @@
 FROM fedora
+
 # Cache layers which will not change
 RUN groupadd teleirc -g 65532 \
-&& useradd -u 65532 -g teleirc -s /bin/bash -M -d /opt/teleirc teleirc
+    && useradd -u 65532 -g teleirc -s /bin/bash -M -d /opt/teleirc teleirc
 
 WORKDIR /opt/teleirc
-CMD ["node", "teleirc.js"]
 
 COPY . /opt/teleirc/
-COPY config.js.example /opt/teleirc/config.js
+COPY config.js /opt/teleirc/config.js
 
 RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo \
-&& dnf -y update --setopt=deltarpm=false \
-&& dnf clean all \
-&& dnf -y install nodejs libicu-devel python gcc-c++ make yarn --setopt=deltarpm=false \
-&& yarn \
-&& dnf -y remove libicu-devel gcc-c++ make \
-&& dnf clean all \
-&& chown -R teleirc:teleirc /opt/teleirc
+    && dnf -y upgrade --setopt=deltarpm=false \
+    && dnf -y install nodejs libicu-devel python gcc-c++ make yarn --setopt=deltarpm=false \
+    && yarn \
+    ; dnf -y remove libicu-devel gcc-c++ \
```

Comment:
what's the reasoning for installing `gcc-c++` and then removing it in the next line? Also, why does this line start with a semicolon instead of continuing with the `&&`?

Reply:
@jwflory ah, makes sense now. That message didn't make that portion clear, but now it is.